src/modules/peek/Peek.Common/Models/IFileSystemItem.cs

```diff
@@ -22,7 +23,45 @@ public interface IFileSystemItem
 
         public string Path { get; init; }
 
-        public IPropertyStore PropertyStore { get; }
+        public uint? Width
+        {
+            get
+            {
+                using DisposablePropertyStore propertyStore = new(PropertyStoreHelper.GetPropertyStoreFromPath(Path));
```

Comment:
Can we wrap that DisposablePropertyStore call into a helper? Would be nice to oneline getting a property from a file. It would keep this class concise.